Automattic/newspack-plugin
# Add Campaign analytics view

### All Submissions:






### Changes proposed in this Pull Request:



Adds a Campaigns analytics view.

Related: #515 

### How to test the changes in this Pull Request:

1. In Newspack->Campaigns view of the plugin, observe a new tab: "Analytics":

![image](https://user-images.githubusercontent.com/7383192/81949516-3cf55200-9603-11ea-8fb0-28697ed7c975.png)

2. Open [Google Analytics web UI](https://analytics.google.com/analytics/web), go to Behavior->Events->Overview and select "Newspack Announcement" event category. You can set a secondary dimension to "Event Action" for a better overview.
3. Compare the data in GA web UI and the plugin view for various date ranges and verify that they match.

### Other information:
